src/SoapCore/TrailingServicePathTuner.cs

```diff
@@ -7,11 +7,16 @@
 
 namespace SoapCore
 {
+    public interface ITrailingServicePathTuner
```

Comment:
What do you think about calling it IServicePathTuner? The trailing-bit could be considered a behavior of the default implementation

Reply:
You are right! PathString is better!